package.json

```diff
@@ -42,13 +42,14 @@
     "dragselect": "^2.7.4"
   },
   "devDependencies": {
-    "@blockly/workspace-backpack": "^5.3.4",
-    "@blockly/dev-scripts": "^3.1.1",
-    "@blockly/dev-tools": "^7.1.6",
-    "blockly": "^10.4.2"
+    "@blockly/workspace-backpack": "^6.0.2",
+    "@blockly/dev-scripts": "^4.0.1",
+    "@blockly/dev-tools": "^8.0.2",
+    "@blockly/plugin-scroll-options": "^6.0.2",
```

Comment:
Don't add the @blockly/plugin-scroll-options dependencies here as we will still use the old test file. Remember to update package-lock.json once you get this removed